lib/timers.js

```diff
@@ -158,7 +158,7 @@ exports.enroll = function(item, msecs) {
 
 // call this whenever the item is active (not idle)
 // it will reset its timeout.
-exports.active = function(item) {
+const active = exports.active = function(item) {
```

Comment:
I've been thinking about it and I think it would be best to only do this for the globals, so only `clearTimeout` in this patch. (Plus we also don't test active/{un}enroll and there's not a terribly good reason to.